src/main/java/org/terasology/blockNetwork/EfficientBlockNetwork.java

```diff
@@ -26,6 +26,10 @@
 import java.util.Iterator;
 import java.util.Set;
 
+/**
+ * Manages networks to represent blocks that connect in a networked manner.
+ * @param <T> The type of node in this network.
```

Comment:
You should add a blank line between the description and the param lists. Do this everywhere